flood/src/main/java/com/dsvl/flood/UdpMsgBuilder.java

```diff
@@ -56,4 +61,23 @@ public static String buildRegisterMsg(String nodeAddress, Integer nodeUdpPort, S
         return MessageFormat.format(REGISTER_MSG_TEMPLATE, String.format("%04d", length), nodeAddress, nodeUdpPort, username);
     }
 
+    /**
+     * Returns a {@code String} of the format {@code length JOIN IP_address port_no}
+     * @param nodeAddress
+     * @param nodeUdpPort
+     * @return the join message
+     */
+    public static String buildJoinMsg(String nodeAddress, Integer nodeUdpPort) {
```

Comment:
a relevant test case should be written.
similar to this https://github.com/KajanM/DSVL/blob/47ab96bd84464c1b7961f142d955fe569b56ade3/flood/src/test/java/com/dsvl/flood/UdpMsgBuilderTest.java#L9-L14